yt/frontends/stream/tests/test_stream_amrgrids.py

```diff
@@ -69,3 +69,23 @@ def load_grids():
         load_amr_grids(grid_data, dims, bbox=bbox, periodicity=(0, 0, 0),
                        length_unit=1.0, refine_by=2)
     assert_raises(YTIllDefinedAMR, load_grids)
+
+
+def test_2665():
```

Comment:
I think it would be helpful if this test name were something other than the issue number. It's ok if the name is long, it's better if it's descriptive!

Reply:
oh yes of course, I did mean to change it before opening the PR !